src/hotspot/share/c1/c1_LIR.cpp

```diff
@@ -483,6 +489,7 @@ void LIR_OpVisitState::visit(LIR_Op* op) {
 
       assert(op1->_info != nullptr, "");  do_info(op1->_info);
       if (op1->_opr->is_valid())       do_temp(op1->_opr); // safepoints on SPARC need temporary register
+      assert(op1->_tmp->is_illegal(), "not used");
```

Comment:
I think safepoints should ideally be implemented as LIR_Op0. SPARC support is removed. We could file a new RFE for that.

Reply:
Well, a grep showed there are more leftovers of sparc and solaris in hotspot, but not a matter of this change.